generated_code/aderdg.py

```diff
@@ -131,3 +151,57 @@ def addNeighbor(self, generator):
   @abstractmethod
   def addTime(self, generator):
     pass
+
+class ADERDGStandard(ADERDGBase):
+  def sourceMatrix(self):
+    return None
```

Comment:
Is there a reason why this isn't in ADERDGBase?

Reply:
Is there a reason why it should be in ADERDGBase?

Reply:
No, you're right.